packages/utils/src/report/get-css-location-from-node.ts

```diff
@@ -0,0 +1,64 @@
+import { ChildNode, Declaration } from 'postcss';
+import { ProblemLocation } from '../types/problems';
+
+type Position = {
+    column: number;
+    line: number;
+}
+
+/**
+ * Set the start and end position for a property value.
+ */
+const setValuePosition = (node: Declaration, start: Position, end: Position) => {
+    const betweenSplit = node.raws.between ? node.raws.between.split('\n') : [''];
+
+    end.line = start.line + betweenSplit.length - 1;
+    end.column = betweenSplit.length === 1 ?
+        // The value is in the same line as the property.
+        start.column + node.prop.length + betweenSplit[0].length + node.value.length :
+        /*
+         * The value is in a different line we need to add one
+         * because the position we are looking for is charecter after the end of the value
+         * e.g.
+         *     value;
+         *          ^
+         */
+        betweenSplit[betweenSplit.length - 1].length + node.value.length + 1;
+
+    start.line = end.line;
+    start.column = end.column - node.value.length;
+};
+
+/**
+ * Returns the location of a CSS node.
+ * @param node postcss Node.
+ * @param isValue Indicates if we need the location of the property value.
+ */
+export const getLocationFromNode = (node: ChildNode, isValue?: boolean): ProblemLocation | undefined => {
```

Comment:
Also I think we should be consistent with what we have right now in [`json-parser`](https://github.com/webhintio/hint/blob/94dfef55140f6aa9b3dc3d071f9558a4afecb47d/packages/utils/src/json-parser.ts#L28) and accept an `options` object ([`IJSONLocationOptions`](https://github.com/webhintio/hint/blob/94dfef55140f6aa9b3dc3d071f9558a4afecb47d/packages/utils/src/types/json-parser.ts#L3)) in case we need to add more things in the future.

Reply:
That type shouldn't be an `interface` :P. I will do it